src/test/java/Problem2Test.java

```diff
@@ -8,7 +8,24 @@
     // Okay to change
     @Test
     public void testBubbleSort() {
-        assertTrue(false);
-        // add your tests here
+
+        int[] input = new int[] {3, 2, 5, 6, 4, 10, 22, 9};
+        int[] input2 = new int[] {3, 2, 5};
+        int[] input3 = new int[] {3,1};
+
+        Problem2.bubbleSort(input);
+        for (int i = 0; i < input.length-1; i++){
+            assertTrue(input[i]<input[i+1]);
```

Comment:
your test verifies the array becomes an increasing order but there are two issues.

1. what if there's duplicates?

2. what if after the sort, the array contains gibberish numbers due to error in sort but it's still in ascending order. in another word, the test need to check the array contains the same numbers as the input too in addition to being sorted